chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/listing/reviewer/DeprecatedFieldReviewer.java

```diff
@@ -0,0 +1,23 @@
+package gov.healthit.chpl.validation.listing.reviewer;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import gov.healthit.chpl.domain.CertifiedProductSearchDetails;
+
+@Component("deprecatedFieldReviewer")
+public class DeprecatedFieldReviewer implements ComparisonReviewer {
+
+    @Autowired
+    public DeprecatedFieldReviewer() {
+    }
+
+    @Override
+    public void review(CertifiedProductSearchDetails existingListing,
+            CertifiedProductSearchDetails updatedListing) {
+        if (!existingListing.getTransparencyAttestationUrl().equals(updatedListing.getTransparencyAttestationUrl())) {
```

Comment:
Can you also add a check for meaingfulUseUserHistory since we deprecated that field without ever doing this sort of check on it?

Reply:
Oh but that check would have to be flag-based because the UI itself is using the deprecated field until the flag is on (I think? double check me on that)